tensorflow_probability/python/distributions/empirical.py

```diff
@@ -217,6 +217,18 @@ def _stddev(self):
     r = samples - tf.expand_dims(self._mean(samples), axis=axis)
     var = tf.reduce_mean(tf.square(r), axis=axis)
     return tf.sqrt(var)
+  
+  def _quantile(self, value, samples=None, **kwargs):
+    if value > 1 or value < 0:
```

Comment:
You don't need to do this check here, it's already handled by the base Distribution class.